package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-log-forwarding",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Let's set the version `2.0.0` as we are changing the major version of serverless

Reply:
Alex want to change a major version after moving to TS